src/base/ParserFaults.messages

```diff
@@ -2959,7 +2952,7 @@ Ends in an error in state: 29. Please report your example at https://github.com/
 
 stmts_term: ID FETCH AND CID LPAREN WITH
 ##
-## Ends in an error in state: 293.
+## Ends in an error in state: 295.
```

Comment:
Annoyingly, I think this now breaks the error message below 
https://github.com/Zilliqa/scilla/blob/c4c860a6f3c69bc22bcaf7d1c6633f6346602856/src/base/ParserFaults.messages#L2963

This is the case for quite a few other cases as well :(

Reply:
It is hard to tell since it is not clear exactly how the state number relates to parser definition.

Reply:
I think its just that if someone does hit this, they'll raise an issue with the wrong error state number (the one they were told by the message). But as mentioned above, I think we can rely on the reported providing the contract anyway, so I don't think its a problem.